### PR TITLE
Add GitHub actions continuous-integration workflow for testing.

### PR DESCRIPTION
**Pull request by csadorf**

### CODE REVIEW

**Comment by csadorf** [comment]
Tests succeeded: https://github.com/csadorf/aiidalab-home/actions/runs/153932207

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Run basic tests for this app on the latest aiidalab-docker image.
+
+name: continuous-integration
+
+on:
+  [push]
```
Comment (CasperWA) [inline .github/workflows/ci.yml:6]: Is there a reason why we're not testing on PRs?
```suggestion
on: [push, pull_request]
```

Comment (csadorf) [reply .github/workflows/ci.yml:6]: No.

**Comment by CasperWA** [comment]
Also here, I wonder why this new Action didn't run for this PR? Should it be `pull_requests` in plural?

**Pull request merged**